sympy/core/tests/test_numbers.py

```diff
@@ -320,6 +320,19 @@ def test_Rational_new():
     assert Rational('.76').limit_denominator(4) == n3_4
     assert Rational(19, 25).limit_denominator(4) == n3_4
     assert Rational('19/25').limit_denominator(4) == n3_4
+    n2_3 = Rational(2, 3)
+    assert Rational('2/3') == n2_3
```

Comment:
Maybe I wasn't clear, but as these do not add anything in terms of testing, I would remove them.

Reply:
Sorry , got confused will remove them now.